tests/test_test_tree_browser.py

```diff
@@ -0,0 +1,13 @@
+import pandas as pd
+
+import adatest._test_tree_browser as ttb
+
+
+class TestIsSubTopic:
+    def test_topic_is_subtopic(self):
+        assert ttb.is_subtopic("/A", "/A/B")
```

Comment:
Internally `is_subtopic()` uses a regular expression, but AFAICT, it boils down to `str.startswith()` ... am I missing something? If so, I need to add a test which shows that case.

Reply:
```python3
is_subtopic("/A", "/AB") -> False
```
whereas
```
"/AB".startswith("/A") -> True)
```

The shared prefix situation can come up in some test trees so we have to explicitly handle this case separately with regular expressions. It might be worth investigating [Pathlib](https://docs.python.org/3/library/pathlib.html) for some nicer handling of these situations in the future. 

+1 on adding a test for it!

Reply:
*bangs head* Yes, of course. I'll definitely add that test.

I agree that PathLib is worth looking at (or even just `os.path.commonpath`). However, I'd suggest we not do that until we have tests running on all the platforms we want to support. This may just be paranoia, but I foresee nasty interactions with the actual OS directory separator.